pysat/tests/test_constellation.py

```diff
@@ -248,8 +241,7 @@ def test_diff_similar_instruments(self):
                                         cost_function)
         diff = results['dummy1']
         dist = results['dist']
-        for i in diff:
-            assert i == 5
+        assert abs(diff - 5).max() == 0
```

Comment:
This is a different test than the one it replaced.  Is there a reason?  Why not use np.all([i == 5 for i in diff])?

Reply:
Should be functionally equivalenet, but will replace with assert np.all(abs(diff - 5)) == 0

Also deleted dist = results, since this is not used in the test